docs/api/genomicRdd.rst

```diff
@@ -152,3 +152,110 @@ Similar to ``transform``/``transformDataset``, there exists a
 ``transmuteDataset`` function that enables transformations between
 ``GenomicRDD``\ s of different types.
 
+Using partitioned Parquet to speed up range based queries
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+GenomicRDDs of types ``AlignmentRecordRDD``, ``GenotypeRDD``, 
+``VariantRDD``, and ``NucleotideFragmentContigRDD`` can be written as Parquet 
+using a Hive-style hierarchical directory scheme that is based on contig and 
+genomic position.  This partitioning reduces the latency of genomic range
+queries against these datasets, which is particularly important for interactive
+applications such as a genomic browser backed by an ADAM dataset.
+
+The genomicRDD function 
+``GenomicRDD.filterByOverlappingRegions(queryRegionsList)`` builds a Spark SQL 
+query that uses this paritioning scheme. This can reduce latencies by more 
+than 20x when repeatedly querying a datset with genomic range filters.   
+On a high coverage alignment dataset, this partitioning strategy improved
+latency from 1-2 minutes to 1-3 seconds when looking up genomics ranges.
+
+**Saving partitioned parquet files to disk**  
+
+A ``GenomicRDD`` can be written to disk as a paritioned Parquet dataset with the
+``GenomicRDD`` function ``saveAsParitionedParquet``. The optional
+``partitionSize`` parameter  defines the width in base pairs of the partitions
+within each contig.
+
+.. code:: scala
+ data.saveAsPartitionedParquet("dataset1.adam", partitionSize = 2000000)
+
+A partitioned dataset can also be created from an input Parquet or SAM/BAM/CRAM
+file using the ADAM ``transformAlignments`` CLI, or Parquet/VCF files using the
+ADAM ``transformGenotypes`` CLI.
+
+**Loading partitioned parquet files**
+
+A GenomicRDD can be loaded from a partitioned {arquet dataset using the
+ADAMContext functions ``loadPartitionedParquetDataset`` or by using the
+``--isBinned`` options in the CLI.
+
+**Layout of Partitioned Parquet directory**
+
+An ADAM partitioned Parquet dataset is written as a three level directory hierarchy.
+The outermost directory is the name of the dataset and contains metadata files as is
+found in unparitioned ADAM Parquet datasets. Within the outer dataset directory,
+subdirectories are created with names based on the each of the genomic contigs found
+in the dataset, for example a subdir will be named ``contig=22`` for chromosome 22.
```

Comment:
is it actually contig or should it be contigName to match our field?

edit, see above
```
+    df.withColumn("positionBin", floor(df("start") / partitionSize))
+      .write
+      .partitionBy("contigName", "positionBin")
```

Reply:
Fixed, it actually is "contigName=22" in the path